### PR TITLE
Transfer benchdocs reorg

### PR DESCRIPTION
**Pull request by Rmalavally**

### CODE REVIEW

File: docs/index.rst
Comment (randyh62) [inline docs/index.rst:1]: I think I would just title this "TransferBench documentation" or something along those lines.

Comment (randyh62) [inline docs/index.rst:1]: I would title this "Using TransferBench" as opposed to "Use TransferBench".

**Review by randyh62 — APPROVED** [review]

File: docs/index.rst
Comment (Rmalavally) [reply docs/index.rst:1]: The header is How to, so 'Using' will be grammatically incorrect. I think we have 'Use' in our template as well. 

https://rocm.docs.amd.com/projects/rocDecode/en/latest/index.html

File: docs/index.rst [hunk 1]
```diff
@@ -1,11 +1,36 @@
-*******************************************
-Welcome to TransferBench's documentation!
-*******************************************
+****************************
+TransferBench documentation
+****************************
 TransferBench is a simple utility capable of benchmarking simultaneous transfers between user-specified devices (CPUs/GPUs).
```
Comment (lpaoletti) [inline docs/index.rst]: Perhaps a little tidying up in the intro:
TransferBench is a utility capable of benchmarking simultaneous transfers between user-specified devices (CPUs/GPUs). A transfer is a single operation where an executor reads and adds values from source (SRC) memory locations, then writes the sum to destination (DST) memory locations. This simplifies to a simple copy operation when dealing with a single SRC/DST.

File: docs/index.rst [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
 A Transfer is defined as a single operation where an executor reads and adds together values from source (SRC) memory locations, then writes the sum to destination (DST) memory locations. This simplifies to a simple copy operation when dealing with single SRC/DST.
 
-The user has control over the SRC and DST memory locations by indicating memory type followed by the device index. TransferBench supports coarse-grained pinned host memory, unpinned host memory, fine-grained host memory, coarse-grained global device memory, fine-grained global device memory, and null memory (for an empty transfer). In addition, the user can determine the size of the transfer (number of bytes to copy) for their tests.
 
-The executor of the transfer can also be specified by the user. The options are CPU, kernel-based GPU, and SDMA-based GPU (DMA) executors. TransferBench also provides the option to choose the number of sub-executors. In case of a CPU executor this argument specifies the number of CPU threads, while for a GPU executor it defines the number of compute units (CU). If DMA is specified as the executor, the sub-executor argument determines the number of streams to be used.
+.. grid:: 2
```
Comment (lpaoletti) [inline docs/index.rst:8]: Should we have a link to the Git repository after the intro?

File: docs/install/install.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
+.. meta::
+  :description: TransferBench documentation 
+  :keywords: TransferBench, API, ROCm, HIP
+
+---------------------------
+TransferBench Installation
+---------------------------
+
```
Comment (lpaoletti) [inline docs/install/install.rst:8]: Shouldn't there be an intro sentence before these two bullets?
Perhaps system requirements?

File: docs/how to/use-transferbench.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+* fine-grained host memory
+* coarse-grained global device memory
+* fine-grained global device memory
+* null memory (for an empty transfer).
+
+In addition, users can determine the size of the transfer (number of bytes to copy) for their tests.
+
+Users can also specify executors of the transfer. The options are CPU, kernel-based GPU, and SDMA-based GPU (DMA) executors. TransferBench also provides the option to choose the number of sub-executors. In case of a CPU executor this argument specifies the number of CPU threads, while for a GPU executor it defines the number of compute units (CU). If DMA is specified as the executor, the sub-executor argument determines the number of streams to be used.
+
+Refer to :ref:`Examples` for a specific example.
```
Comment (lpaoletti) [inline docs/how to/use-transferbench.rst]: We only have one example. This bounces the user to a page with a single link. 
Let's get rid of the page and have a new link. 
Something like:
See ConfigFile Format for an example of using TransferBench.

**Review by lpaoletti — COMMENTED** [review]
The topic is called:
ConfigFile Format

However, the TOC is named:
Configfile format

They should align. 

============================
Taking a quick look at the Intro paragraph:
A Transfer is defined as a single operation where an Executor reads and adds together values from Source (SRC) memory locations, then writes the sum to destination (DST) memory locations. This simplifies to a simple copy operation when dealing with single SRC/DST.:

-paragraph ends with a period/colon
-Transfer uses an initial cap
-does this paragraph actually introduce what the topic is about? is this a tutorial?

File: docs/index.rst (hunk 1 above)
Comment (Rmalavally) [reply docs/index.rst]: Fixed

File: docs/index.rst (hunk 2 above)
Comment (Rmalavally) [reply docs/index.rst:8]: Fixed.

File: docs/install/install.rst (hunk 3 above)
Comment (Rmalavally) [reply docs/install/install.rst:8]: We need the content from the Component team.

File: docs/how to/use-transferbench.rst (hunk 4 above)
Comment (Rmalavally) [reply docs/how to/use-transferbench.rst]: Done.

**Comment by Rmalavally** [comment]
Fixed.

File: docs/how to/use-transferbench.rst (hunk 4 above)
Comment (Rmalavally) [reply docs/how to/use-transferbench.rst]: Thanks, Leo. I added the example to the Use TransferBench file for contextual reference.

**Review by gilbertlee-amd — APPROVED** [review]

**Pull request merged**